tools/ms2snoop/MS2snoop.xml

```diff
@@ -1,117 +1,215 @@
-    --min_number_scan '$min_number_scan'
-;
-    ]]></command>
+
+    #if $param.version:
+
+        --version
+
+    #else
+
+        --output '$frag_result_txt'
+        --compounds '$compound_txt'
+        --fragments '$peaklist_frag_tsv'
+        --precursors '$peaklist_preco_tsv'
+        --tolmz '$param.tolmz'
+        --tolrt '$param.tolrt'
+        --seuil_ra '$param.seuil_ra'
+        --mzdecimal '$param.mzdecimal'
+        --r_threshold '$param.r_threshold'
+        --min_number_scan '$param.min_number_scan'
+
+        #if $param.debug:
+            --debug
+        #end if
+
+        #if $param.verbose:
+            --verbose
+        #end if
+
+    #end if
+
+        ]]>
+    </command>
 
     <inputs>
-        <param
-            type="data"
-            format="tabular"
-            name="compound_txt"
-            label="compounds_pos.txt"
-        />
-        <param
-            type="data"
-            format="tabular"
-            name="peaklist_frag_tsv"
-            label="peaklist_fragments.tsv"
-        />
-        <param
-            type="data"
-            format="tabular"
-            name="peaklist_preco_tsv"
-            label="peaklist_precursors.tsv"
-        />
-        <param
-            name="tolmz"
-            type="float"
-            min="0.0001"
-            max="10"
-            value="0.01"
-            label="MZ Tolerence"
-        />
-        <param
-            name="tolrt"
-            type="integer"
-            min="0"
-            max="30"
-            value="20"
-            label="RT Tolerence"
-        />
-        <param
-            name="seuil_ra"
-            type="float"
-            min="0"
-            max="1"
-            value="0.05"
-            label="r pearson correlation threshold between precursor and fragment absolute intensity"
-        />
-        <param
-            name="mzdecimal"
-            type="integer"
-            min="0"
-            max="5"
-            value="0"
-            label="nb decimal for mz"
-        />
-        <param
-            name="r_threshold"
-            type="float"
-            min="0"
-            value="0.85" label="fragment absolute intensity"
-        />
-        <param
-            name="min_number_scan"
-            type="integer"
-            min="0"
-            max="25"
-            value="8"
-            label="Present in at least X scan"
-            help="fragments are kept if they are found in a minimum number of scans"
-        />
+
+        <conditional name="param">
+
+            <param
+                type="boolean"
+                value="false"
+                label="Prints version"
+                argument="--version"
+                help="The tool prints its version and then exits"
+            />
+
+            <when value="false">
+
+                <param
+                    type="data"
+                    format="tabular"
+                    name="compound_txt"
+                    label="compounds_pos.txt"
```

Comment:
the label should be a description and not a file name

Reply:
fixed!